### PR TITLE
Add atool2

### PR DESCRIPTION
**Pull request by jonashaag**
I don't know why we need this

```
/configure --prefix=$BUILD_PREFIX
```

but it works. It doesn't work with

```
/configure --prefix=$PREFIX
```

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jonashaag** [comment]
@conda-forge/staged-recipes ready for review

**Comment by jonashaag** [comment]
@conda-forge/help-perl

**Comment by jonashaag** [comment]
@conda-forge/help-perl

File: recipes/atool2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/solsticedhiver/atool2/archive/{{ version }}.tar.gz
+  sha256: 87fc55cc21e2ac74c356797fc4e1d73b37dd37a6b91811cea778c68787079239
+
+build:
+  skip: true  # [win] # Missing automake
```
Comment (ocefpaf) [inline recipes/atool2/meta.yaml:13]: Just to make it easier to read and not confuse with the pre-processor selector.

```suggestion
  # Missing automake
  skip: true  # [win]
```

File: recipes/atool2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  url: https://github.com/solsticedhiver/atool2/archive/{{ version }}.tar.gz
+  sha256: 87fc55cc21e2ac74c356797fc4e1d73b37dd37a6b91811cea778c68787079239
+
+build:
+  # Missing automake
+  skip: true  # [win]
+  number: 0
+  # noarch: generic # Re-enable after fixing Windows build
```
Comment (ocefpaf) [inline recipes/atool2/meta.yaml:16]: Note that noarch packages are built on Linux and installed on macOS and Windows. What is happening with the Windows build here? Anything that is arch specific won't allow this to become a noarch package later.

Comment (jonashaag) [reply recipes/atool2/meta.yaml:16]: Could it be the case that `noarch: generic` packages are not built on Linux? This is the failed pipeline https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=991382&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4

Comment (ocefpaf) [reply recipes/atool2/meta.yaml:16]: It is missing automake:

```
Encountered problems while solving:
  - nothing provides requested automake
```

> Could it be the case that noarch: generic packages are not built on Linux?

They are built only on Linux in the feedstock. If you are sure this can be noarch generic, we can ignore that Windows failure and make it noarch here. We just need to be sure there are no compiled stuff there and that the files will be placed in sane directories for all OSes.

Comment (ocefpaf) [reply recipes/atool2/meta.yaml:16]: The original `atool` seems to be a perl script dropped in a bin directory, right? Did you ever try to use that on Windows? My guess is that it won't work b/c it should be in the `%PREFIX%\Library\bin`. On Linux and mac that is `$PREFIX/bin`.

TL;DR I'm not sure how to best make this noarch generic. You can try OS-noarch and build it once for Linux+mac and they another build for Windows. However, all that won't matter here b/c your Windows build is not working. Can you propagate this comment to an issue in the feedstock. We can merge this as-is for now.
